src/global/theme.tsx

```diff
@@ -25,7 +19,9 @@ enum Theme {
   DARK = 'dark',
 }
 
-export const ThemeContext = createContext<{theme: Theme.LIGHT | Theme.DARK}>({theme: Theme.LIGHT});
+export const ThemeContext = createContext<{theme: Theme.LIGHT | Theme.DARK; passToPopups?: boolean}>({
```

Comment:
I believe, we need to deprecate `passToPopups` in 8.0 and promote overwriting top theme via `ThemeProvider` instead